loro-js/tests/richtext.test.ts

```diff
@@ -300,6 +300,27 @@ describe("richtext", () => {
     expect(text.toDelta()).toStrictEqual([
       { insert: "你b"},
     ]);
+  });
+
+  it("Slice", () => {
```

Comment:
Add a new test that slices on emoji? Some emojis contain code points that need 2 UTF-16 chars to encode.